src/backend/defs/rest-api/paths/import.yaml

```diff
@@ -0,0 +1,11 @@
+contacts_import:
+  post:
+    consumes:
+    - text/vcard
+    - text/vcf
+    - text/x-vcard
+    responses:
+      '200':
+        description: Import successful
+      '400':
+        description: Syntax error
```

Comment:
Miss : 
 * "security" directive
 * "produces" directive
 * "description" & "schema" for responses.

I suggest the route to be `/v1/contacts/import` or  `/v1/import/contacts`